src/components/forms/ReadOnlyField.tsx

```diff
@@ -11,7 +12,7 @@ interface Props {
 }
 
 const ReadOnlyField: FunctionComponent<Props> = (props) => {
-  if (!props.children) {
+  if (!getAttributeValue(props, 'children')) {
```

Comment:
This opts out of the TypeScript type system be enforcing types (because `props` is now any). Instead, ensure that `props` is correctly typed.